### ISSUE TITLE
[no-missing-import/-require] Support `node:` URL scheme

### ISSUE THREAD

**Issue by valtlai** [mysticatea/eslint-plugin-node#275]
The `node/no-missing-import` and `node/no-missing-require` rules don’t support the [`node:` URL scheme](https://nodejs.org/api/esm.html#esm_node_imports), so there’s a false positive in cases like the following:

```js
import fs from 'node:fs';

// or

const fs = require('node:fs');
```

Currently, the rules conflict with [`unicorn/prefer-node-protocol`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-node-protocol.md).

**Comment by crystalfp** [comment]
also `node/no-missing-import` does not accept `import fs from "node:fs"`

**Comment by valtlai** [comment]
I actually meant `no-missing-import`/`-require` and not `no-extraneous-import`/`-require`. Updated the title and comment.
